src/+datools/+statistical/+ensemble/Hybrid.m

```diff
@@ -37,7 +37,7 @@ function analysis(obj, R, y)
             X = obj.Ensemble;
             alphas = obj.Alphas;
             
-            Xf = X;
+            xf = X;
```

Comment:
I don't understand why this was made lowercase. I know matlab variables are lowercase, but in our notation (which ALWAYS overrides matlab notation) Xf stands for an ensemble, and xf stands for a particular realization (so a vector not a matrix). Please change this back.

Reply:
It was giving an error when i used Xf saying that xf is not defined. Let me take another look at it

Reply:
xf has been used in all the filter classes to denote the total forecast ensemble. Do you want me to change just for this filter or for every other filters too?

Reply:
I think amit did that... zzz. Whatever it's fine